src/screens/AnonymousChatScreen/index.tsx

```diff
@@ -67,9 +67,9 @@ const AnonymousChatScreen = () => {
     goToChatInfoScreen,
     sendChat,
     selfAnonUserInfo,
-    updateChatContinuity,
     scrollContext,
-    flatListRef: scrollRef
+    flatListRef: scrollRef,
+    updateChatContinuity
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateChatContinuity` function is now assigned separately rather than being destructured. This change doesn't seem to affect the functionality, but it's important to ensure that this function is still being called correctly in the rest of the code.
```diff
-    updateChatContinuity,
     scrollContext,
     flatListRef: scrollRef
+    updateChatContinuity
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->